deployments/gpu-operator/charts/node-feature-discovery/templates/clusterrole.yaml

```diff
@@ -80,7 +94,7 @@ rules:
   - update
 {{- end }}
 
-{{- if and .Values.gc.enable .Values.gc.rbac.create (or .Values.enableNodeFeatureApi .Values.topologyUpdater.enable) }}
+{{- if and .Values.gc.enable .Values.gc.rbac.create (or (and .Values.featureGates.NodeFeatureAPI .Values.enableNodeFeatureApi) .Values.topologyUpdater.enable) }}
```

Comment:
is `featureGates.NodeFeatureAPI` enabled by default? Otherwise we have change in behavior when enableNodeFeatureApi is set.